foundry.toml

```diff
@@ -1,9 +1,35 @@
 [profile.default]
-src = 'src'
-out = 'out'
-libs = ['lib']
+  libs = ['lib']
+  optimizer = true
+  optimizer_runs = 10_000_000
+  out = 'out'
+  solc_version = "0.8.16"
+  src = 'src'
+  verbosity = 3
+
+[profile.ci]
+  fuzz = { runs = 5000 }
+  invariant = { runs = 1000 }
+
+[profile.lite]
+  fuzz = { runs = 50 }
+  invariant = { runs = 10 }
+  # Speed up compilation and tests during development.
+  optimizer = false
 
 [rpc_endpoints]
-optimism = "${OPTIMISM_RPC_URL}"
+  optimism = "${OPTIMISM_RPC_URL}"
+
+[fmt]
+  bracket_spacing = false
+  int_types = "long"
+  line_length = 100
+  multiline_func_header = "attributes_first"
+  number_underscore = "thousands"
+  quote_style = "double"
+  single_line_statement_blocks = "single"
+  tab_width = 2
+  wrap_comments = true
 
-# See more config options https://github.com/foundry-rs/foundry/tree/master/config
+  # Copied from https://github.com/ScopeLift/foundry-template/blob/main/foundry.toml
```

Comment:
Same nit, we can delete this. Normally you'd create the repo from the template which wouldn't have these comments. Feel free to resolve this comment yourself.

Reply:
I actually think it would be nice to include a link to our template in the template, so that when we create new projects from it there is a record of where the code came from. This is the kind of thing that I think could easily be forgotten ("where did these settings come from??"), especially in a long-lived project